TopReadWidget/WMFTodayTopReadWidgetViewController.swift

```diff
@@ -5,14 +5,9 @@ import WMF
 class WMFTodayTopReadWidgetViewController: ExtensionViewController, NCWidgetProviding {
     
     // Model
-    var siteURL: URL!
     var groupURL: URL?
+    var siteURL: URL?
```

Comment:
To simplify, is there a benefit to having this in a property? Every context I've found also has a `dataSource`, maybe you could pull it from there each time.